tests/phpunit/tests/term.php

```diff
@@ -308,4 +308,46 @@ public function test_orphan_category() {
 		$cat_id2 = self::factory()->category->create( array( 'parent' => $cat_id1 ) );
 		$this->assertWPError( $cat_id2 );
 	}
+
+
+	/**
+	 * @ticket 58329
+	 * @dataProvider data_get_term_filter
+	 */
+	public function test_get_term_filter( $filter ) {
+		$cat_id1 = self::factory()->category->create();
+
+		$term = get_term( $cat_id1, '', OBJECT, $filter );
+
+		$this->assertSame( $filter, $term->filter, 'Sanitize should match' );
+	}
+
+	/**
+	 * @ticket 58329
+	 * @dataProvider data_get_term_filter
+	 */
+	public function test_get_term_filtered( $filter ) {
+		$cat_id1 = self::factory()->category->create();
+		$cat     = self::factory()->category->create_and_get();
+		add_filter(
+			'get_term',
+			function() use ( $cat ) {
```

Comment:
```suggestion
			static function () use ( $cat ) {
```

Per [Changeset 56559](https://core.trac.wordpress.org/changeset/56559), Include one space after function keyword for closures and use `static`